src/main/java/net/atomique/ksar/Main.java

```diff
@@ -18,13 +26,22 @@
 public class Main {
 
   private static final Logger log = LoggerFactory.getLogger(Main.class);
-
+  private static boolean gui = true;
   static Config config = null;
   static GlobalOptions globaloptions = null;
   static ResourceBundle resource = ResourceBundle.getBundle("net/atomique/ksar/Language/Message");
 
   public static void usage() {
-    show_version();
+    log.info("-input : input sar statistics file \n");
+    log.info("-n : toggles nongui mode for exporting\n");
+    log.info("-outputCSV : location of parsed CSV file\n");
+    log.info("-outputPDF : location of output pdf file\n");
+    log.info("-outputIMG : prefix for output images\n");
+    log.info("-width : width for output png charts\n");
+    log.info("-heigth : heigth for output png charts");
```

Comment:
I was refering to previous version of kSar where console mode was working

It was following
-width <size> : make JPG/PNG with specified width size (default: 800)
-height <size> : make JPG/PNG with specified height size (default: 600)

BTW there is full output for help in older version

-version: show kSar version number
-help: show this help
-input <arg>: argument must be either ssh://user@host/command or cmd://command o
r file://path/to/file or just /path/to/file
-graph <graph list>: space separated list of graph you want to be outputed
-showCPUstacked: will make the CPU used graph as stacked
-showMEMstacked: will make the Memory graph as stacked (linux only)
-cpuFixedAxis:  will graph CPU used with fixed axis from 0% to 100%
-showIntrListstacked : will make the Interrupt List graph as stacked
-showTrigger:  will show trigger on graph (disabled by default)
-noEmptyDisk: will not export disk with no data
-tile: will tile window
-userPrefs: will use the userPrefs for outputing graphs (last export of this hos
t)
-showOnlygraphName: will only print graph name available for that data (to be us
e for -graph)
-addHTML: will create an html page with PNG/JPG image
-outputPDF <pdf file> : output the pdf report to the pdf file
-outputPNG <base filename> : output the graphs to PNG file using argument as bas
e filename
-outputJPG <base filename> : output the graphs to JPG file using argument as bas
e filename
-outputCSV <CSV file> : output the CSV file
-width <size> : make JPG/PNG with specified width size (default: 800)
-height <size> : make JPG/PNG with specified height size (default: 600)
-startdate <date> : will graph the range begining at that time
-enddate <date> : will graph the range until that date
-solarisPagesize <pagesize in B>: will set solaris pagesize
-wizard: open with unified login popup
-replaceShortcut <xml file>: replace all shortcut with those in the xml file
-addShortcut <xml file>: add shortcut from the xml file
-startup: open window marked for opening at startup